recapupload.py

```diff
@@ -27,7 +27,6 @@
 
 from bigcases_settings import settings
 
-
```

Comment:
`recapupload.py` is imported from my repo (https://github.com/johnhawkinson/recapupload/) and passes PEP-8 without these changes. I would rather they be reverted, therefore, to minimize unnecessary version skew with the upstream.